src/components/utils/include/utils/file_system.h

```diff
@@ -241,6 +241,18 @@ bool WriteBinaryFile(const std::string& name,
  */
 bool ReadBinaryFile(const std::string& name, std::vector<uint8_t>& result);
 
+/**
+ * @brief Reads from file
+ *
+ * @param name path to file
+ * @param result read data
+ * @param offset number of bytes to be ignored from begining of file
+ * @return returns true if the operation is successfully.
+ */
+bool ReadBinaryFile(const std::string& name,
```

Comment:
This should have a `length` parameter to determine how many bytes to read, currently it just reads all of the bytes after `offset`